app/services/articles/array_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Articles
+  module ArrayHelper
```

Comment:
I'm not a big fan of this module, it's only used in one place. Instead of moving this here, how about removing both methods in `feed.rb` and replacing them with

```ruby
def shuffle_results(results)
  first_half = results.first(results.length / 2)
  last_half = results.drop(first_half.size)
  first_half.shuffle + last_half.shuffle
```

Then you can replace all occurrences of `ArrayHelper.first_half(results).shuffle + ArrayHelper.last_half(results)` with `shuffle_results(results)` or even get rid of the intermediate variables, i.e.

```ruby
shuffle_results(more_experience_level_weight_experiment)
```

Reply:
I agree with @citizen428, I don't tyink we need a helper (which in the context of Rails it should be a view helper) just for splitting an array in two